syntax = "proto3";

package Pos_msg;

message pos320{
    int32 id = 1;
    int32 length = 2;
    int32 mode = 3;
    int32 time1 = 4;
    int32 time2 = 5;
    int32 num = 6;
    float lat = 7;
    float lon = 8;
    float height = 9;
    float v_n = 10;
    float v_e = 11;
    float v_earth = 12;
    float roll = 13;
    float pitch = 14;
    float head = 15;
    float a_n = 16;
    float a_e = 17;
    float a_earth = 18;
    float v_roll = 19;
    float v_pitch = 20;
    float v_head = 21;
    int32 status1 = 22;
    int32 status2 = 23;
    int32 checksum = 24;
}
